sycl/include/sycl/ext/oneapi/bfloat16.hpp

```diff
@@ -60,12 +65,15 @@ class bfloat16 {
     float *res = reinterpret_cast<float *>(&y);
     return *res;
 #else
-    return __spirv_ConvertBF16ToFINTEL(a);
+    return __devicelib_ConvertBF16ToFINTEL(a);
```

Comment:
There is another call to `__spirv_ConvertBF16ToFINTEL` on line 120 of this file, which should also be changed.

Reply:
Yes, now fixed.